libavcodec/h26x/h2656_inter_template.c

```diff
@@ -0,0 +1,572 @@
+/*
```

Comment:
h2656_inter mismatched with h2656_mc.asm
better to use h2656_mc_template.c if you do not want change asm name.

Reply:
inter is better. I will update them to inter.